src/Microsoft.ML.Data/Training/TrainerEstimatorBase.cs

```diff
@@ -132,21 +132,16 @@ protected virtual void CheckLabelCompatible(SchemaShape.Column labelCol)
         protected TTransformer TrainTransformer(IDataView trainSet,
             IDataView validationSet = null, IPredictor initPredictor = null)
         {
-            var cachedTrain = Info.WantCaching ? new CacheDataView(Host, trainSet, prefetch: null) : trainSet;
```

Comment:
Does this remove auto-caching from just the estimators API, or from the application level, like maml.exe?

Reply:
Mainly for estimators but I expect that in the future, users need to call a specific transform to cache an `IDataView` (mentioned in the original issue).